### PR TITLE
[PP] adding div and sub operations 

### PR DESCRIPTION
**Pull request by anton-potapov**
For mean value preprocessing to function (in near future) following were added
### Details:
 - `div` operation added
 - `sub` operation added
 - unit tests

@anna-khakimova , @dmatveev FYI

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
Why do we need this?

**Comment by ilyachur** [comment]
As I know at the current moment IE plugins don't use GAPI for mean scale.

**Comment by anna-khakimova** [comment]
> Why do we need this?

cc @dmatveev

**Comment by anna-khakimova** [comment]
> As I know at the current moment IE plugins don't use GAPI for mean scale.

cc @dmatveev

**Comment by anton-potapov** [comment]
> As I know at the current moment IE plugins don't use GAPI for mean scale.

This is correct. In order this to change, and  (GAPI) PreProcessing to support mean scale preprocessing some basic blocks is needed.  These operations are those basic blocks needed to implement the logic.

File: inference-engine/tests_deprecated/fluid_preproc/cpu/fluid_tests_cpu.cpp [hunk 1]
```diff
@@ -183,6 +183,20 @@ INSTANTIATE_TEST_CASE_P(ConvertDepthFluid, ConvertDepthTestGAPI,
...
+                                Values(TEST_SIZES),
+                                Values(cv::Scalar{0.229}),
+                                Values(0)));
+
+INSTANTIATE_TEST_CASE_P(SubCFluid, SubCTestGAPI,
+                        Combine(Values(CV_32F),
+                                Values(1),      //channels
+                                Values(TEST_SIZES),
+                                Values(cv::Scalar{0.229}),
+                                Values(0)));
```
Comment (anna-khakimova) [inline inference-engine/tests_deprecated/fluid_preproc/cpu/fluid_tests_cpu.cpp:198]: Please relax tolerance to 0.00001 value.

File: inference-engine/tests_deprecated/fluid_preproc/cpu/fluid_tests_cpu.cpp [hunk 2]
```diff
@@ -284,6 +298,11 @@ INSTANTIATE_TEST_CASE_P(Reorder_CHW2HWC, ColorConvertTestIE,
                                 Values(TEST_SIZES),
                                 Values(0)));
 
+INSTANTIATE_TEST_CASE_P(MeanValueGAPI32F, MeanValueGAPI,
+                        Combine(//Values(CV_8U, CV_32F),
+                                Values(TEST_SIZES),
+                                Values(0)));
```
Comment (anna-khakimova) [inline inference-engine/tests_deprecated/fluid_preproc/cpu/fluid_tests_cpu.cpp:304]: Please relax tolerance to 0.00001 value.

File: inference-engine/tests_deprecated/fluid_preproc/cpu/fluid_tests_cpu.cpp [hunk 3]
```diff
@@ -284,6 +298,11 @@ INSTANTIATE_TEST_CASE_P(Reorder_CHW2HWC, ColorConvertTestIE,
                                 Values(TEST_SIZES),
                                 Values(0)));
 
+INSTANTIATE_TEST_CASE_P(MeanValueGAPI32F, MeanValueGAPI,
+                        Combine(//Values(CV_8U, CV_32F),
```
Comment (anna-khakimova) [inline inference-engine/tests_deprecated/fluid_preproc/cpu/fluid_tests_cpu.cpp:302]: Please remove comment.

**Review by anna-khakimova — CHANGES_REQUESTED** [review]

**Comment by anna-khakimova** [comment]
Please rebase the PR.

**Pull request closed without merging**